### ISSUE TITLE
Post Removal Request - UW Homework Solutions

### ISSUE THREAD

**Issue by jsanders89** [kayahr/wlandsuite#1]
Hello,

My name is Jonathan and I work with the introductory computer science courses at the University of Washington. The following post in this repository contains solution code to an assignment based on our Huffman homework assignment:

https://github.com/kayahr/wlandsuite/blob/master/src/main/java/de/ailis/wlandsuite/huffman/HuffmanTree.java

This post's solution code is indexed by Google, enabling current students to easily find it and cheat on their assignments.

Would you please remove this post or make it private in order to help us enforce our academic policy?

By removing the post, you would be removing a tool that our students could use in academic misconduct.

Thank you,
Jonathan Sanders
Intro Support, Computer Science & Engineering
University of Washington
jsanders@cs.washington.edu

More info on our course: http://courses.cs.washington.edu/courses/cse142/15su/

**Comment by kayahr** [comment]
I'm not one of your students and this source code is not a part of some _homework_. It is part of an Open Source project. This means the file has to be publicly available for everyone. Even your students. So I have to decline your request.

I don't know what keywords you used to find my source code. I tried multiple combinations of `huffman`, `java`, `tree`, `algorithm` and `compression` and I found dozens of other source code examples. Lot of them located on servers of US universities. Are you going to take them all down? I think this is a battle you can't win. Maybe you should spend the time teaching your students properly instead so they  don't need to cheat on such a simple topic.

**Comment by jsanders89** [comment]
Hey Klaus,

Thank you for your response.  I made a mistake with your program, thinking it was a solution to an assignment from another school when it was not.  

If you are still curious, the way I found your program was by searching this on google:
"public class HuffmanTree" HuffmanNode BitInputStream BitOutputStream

Your program was the first result.

I am not trying to remove all huffman coding examples from the internet, just the ones that are solutions to our specific assignment that is used by multiple universities across the US.  I've had more luck than you might think, and it has made a tangible difference for our classes.

Thanks again for your time.

Best,
Jonathan
